backend/app.js

```diff
@@ -0,0 +1,35 @@
+const express = require("express");
+const mongoose = require("mongoose");
+const bodyParser = require("body-parser");
+const passport = require("passport");
+require("dotenv").config();
+
+// const users = require("./routes/user");
```

Comment:
Do we need this?

Reply:
No it's old code we'll remove it in clean up, the passport code is currently just there to give an example of how we can use redux and redux-thunk